host.py

```diff
@@ -319,19 +319,23 @@ def run(
         *,
         env_extra: Optional[Mapping[str, Optional[str]]] = None,
         cwd: Optional[str] = None,
+        sudo: Optional[bool] = None,
```

Comment:
1. why do we want sudo as another argument to the run function? Now the function call will look like: `h.run(..., true).`  I don't know what `true)` does until I look at run. That's more work and harder to read than `h.run("sudo..."`)

Reply:
Note the `*`. You cannot write `h.run(..., true)`. You are forced to use keyword arguments, you would have to write `h.run(..., sudo=true)`, which seems pretty clear.

You still run run `h.run("sudo...")`. However, that will not work together with a `env_extra` argument because `sudo` won't pass the environment variables on.